html/views/common.js

```diff
@@ -9,7 +9,7 @@ function jammer(k, v) {
 	var inJam = isTrue(WS.state["ScoreBoard.InJam"]);
 
 	if (jammerName == null || jammerName == "") {
-		jammerName = leadJammer ? "Lead" : "";
+		jammerName = leadJammer ? "" : "";
```

Comment:
Will this break the main scoreboard when there's a lead jammer we don't know the name of?

Reply:
You are right. I will put it back

Reply:
But it would be nice to not have the "Lead" name on the overlay.

Reply:
I'm not that expert in JS. Would there be a posibility to check if the function is called from the overlay wihtout duplicating the whole function and call that one out of the overlay?

Reply:
This isn't a JS thing, it's CRG specific. This particular function is also kinda hairy, and has been the source of a string of bugs.

So I'd like to keep the logic together, so we don't inadvertently break the overlay when changing the scoreboard display.

I'd suggest a small wrapper function to pass in a bool. Though at this stage we probably should be looking at using WS.Register rather than continuing to abuse sbModify.

Reply:
I moved this to the overlay index.js and used WS.Register instead of sbModify to display Lead,Starpass and Skaternames